serverlessworkflow/modules/ROOT/pages/persistence/postgresql-flyway-migration.adoc

```diff
@@ -0,0 +1,51 @@
+= Migrate Your Postgresql database with Kogito upgrade
+
+Kogito provides out of the box way to manage your database schema changes with each upgrade. Kogito uses flyway to achieve the same functionality.
+
+When you upgrade your Kogito version, by default it will pick up the latest changes available from Kogito and apply them automatically.
+
+== How to migrate
+
+.Ways to Migrate your Kogito database with Kogito and Flyway
+* Kogito provides a default mechanism for migrating your db while updating the kogito version using `quarkus.flyway.migrate-at-start` (default `true`). This would create a schema history table `flyway_schema_history` in your DB to track the version of each database, recording in it every versioned migration file applied to build that version.
+
+* If you want to migrate manually you can use flyway migration CLI tool.
+** Set the property `quarkus.flyway.migrate-at-start` as `false`.
+** Download flyway migration command-line tool from link:https://flywaydb.org/download/community[here].
+** Add the following properties in the /conf/flyway.conf
++
+--
+[source,console]
+----
+flyway.url=jdbc:postgresql://localhost:5432/foobardb
+flyway.user=foo
+flyway.password=bar
+----
+--
+** You can specify these options with commands as well. If flyway CLI will prompt for username and passwords if the missing in configuration.
++
+--
+[source,console]
+----
+$ flyway migrate -url=jdbc:postgresql://localhost:5432/foobardb -user=foo -password=bar
+----
+--
+** You can specify location of sql files which needs to be migrated using the flyway.locations option. For example,
++
+--
+[source,console]
+----
+flyway.locations=classpath:com.mycomp.migration,database/migrations,filesystem:/sql-migrations,s3:migrationsBucket,gcs:migrationsBucket
```

Comment:
This also belongs to the CLI migration path, right? For Quarkus way we should probably include it as `quarkus.flyway.locations`

Reply:
@akumar074 I guess it should be mentioned here how to manually download the sql scripts. Link to artifact, so user can extract and apply the changes